pkg/controller/stacks/install/installjob.go

```diff
@@ -52,7 +52,7 @@ const (
 
 var (
 	jobBackoff                = int32(0)
-	registryDirName           = ".registry"
```

Comment:
I think the relative paths were needed for the virtual filesystem based unit tests (afero).

Reply:
No other concerns If this works.

Reply:
k - I'll have to look into it

Reply:
It turns out this wasn't needed by afero-based unpack unit tests (they only use the `/ext-pkg/` side), but also, my changes were breaking the unpack unit tests. I've fixed the broken behavior, and I've added a new test.

Reply:
Resolving this